### PR TITLE
Monetization (MON-394): SitemapPage - add unit tests

### PR DESCRIPTION
**Pull request by saipetch**

### CODE REVIEW

File: extensions/wikia/SitemapPage/SitemapPageModel.class.php [hunk 1]
```diff
@@ -10,16 +10,7 @@ class SitemapPageModel extends WikiaModel {
...
-		WikiFactoryHub::VERTICAL_ID_OTHER => 'Other',
-		WikiFactoryHub::VERTICAL_ID_TV => 'TV',
-		WikiFactoryHub::VERTICAL_ID_VIDEO_GAMES => 'Games',
-		WikiFactoryHub::VERTICAL_ID_BOOKS => 'Books',
-		WikiFactoryHub::VERTICAL_ID_COMICS => 'Comics',
-		WikiFactoryHub::VERTICAL_ID_LIFESTYLE => 'Lifestyle',
-		WikiFactoryHub::VERTICAL_ID_MUSIC => 'Music',
-		WikiFactoryHub::VERTICAL_ID_MOVIES => 'Movies',
-	];
+	protected $verticals = null;
```
Comment (jogura) [inline extensions/wikia/SitemapPage/SitemapPageModel.class.php]: Any harm in just initializing `$verticals` here and remove the need for the `getVerticals()` function below?

File: extensions/wikia/SitemapPage/tests/SitemapPageTest.php [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+			->method( 'getLimitPerPage' )
+			->will( $this->returnValue( $limitPerPage ) );
+		$limit = $mock->getLimitPerList( $level );
+		$this->assertEquals( $exp, $limit );
+	}
+
+	public function getLimitPerListDataProvider() {
+		return [
+			[ 1, 10000],
+			[ 2, 100]
```
Comment (jogura) [inline extensions/wikia/SitemapPage/tests/SitemapPageTest.php]: minor: formatting

Comment (jogura) [inline extensions/wikia/SitemapPage/tests/SitemapPageTest.php]: Any tests for level 0 or a negative level? Level 3, 4, 5, etc? Edge/corner cases?

**Comment by jogura** [comment]
LGTM, merge when all the tests pass.
